develop/tutorials/articles/192-item-selector/03-customizing-item-selector-views.markdown

```diff
@@ -0,0 +1,390 @@
+# Creating Custom Selection Views for the Item Selector [](id=creating-custom-selection-views-for-the-item-selector)
```

Comment:
Hey @mwilliams2014 DisplayContext are not mandatory to create an Item Selector View. We started to follow the pattern of creating DisplayContext some time ago as a way to extract logic from jsps to java classes. However, this is not mandatory and it's up to the developer to decide how they want to implement the view and the markup.